autodoc-nav/docs_nav_2.1.x.yml.head.in

```diff
@@ -0,0 +1,116 @@
+- title: Getting Started
+  icon: /assets/images/icons/documentation/icn-quickstart-color.svg
+  items:
+    - text: Overview
+      url: /../../getting-started-guide/latest/overview
+    - text: Prepare to Administer
+      url: /../../getting-started-guide/latest/prepare
+    - text: Expose your Services
+      url: /../../getting-started-guide/latest/expose-services
+    - text: Protect your Services
+      url: /../../getting-started-guide/latest/protect-services
+    - text: Improve Performance
+      url: /../../getting-started-guide/latest/improve-performance
+    - text: Secure Services
+      url: /../../getting-started-guide/latest/secure-services
+    - text: Set Up Intelligent Load Balancing
+      url: /../../getting-started-guide/latest/load-balancing
+
+- title: Guides &amp; References
+  icon: /assets/images/icons/documentation/icn-references-color.svg
+  items:
+    - text: Quickstart Guide
+      url: /getting-started/introduction
+      items:
+    - text: Five-minute quickstart
+      url: /getting-started/quickstart
+
+    - text: Configuring a Service
+      url: /getting-started/configuring-a-service
+
+        - text: Configuring a gRPC Service
+          url: /getting-started/configuring-a-grpc-service
```

Comment:
@bungle Looks like the builds are breaking because this is indented too far.

Reply:
@lena-larionova, thank you for pointing me on this, it is now fixed.